client/allocator/src/freeing_bump.rs

```diff
@@ -331,6 +330,16 @@ pub struct FreeingBumpHeapAllocator {
 	free_lists: FreeLists,
 	total_size: u32,
 	poisoned: bool,
+	#[cfg(feature = "track-heap")]
+	/// Tracks maximum of `(total_size, bumper)`.
+	tracker: (u32, u32),
```

Comment:
It should be much more hassle to add a dedicated nominal type for this.

Reply:
I wanted to say the same :P